source/collimation.f90

```diff
@@ -2347,29 +2329,18 @@ subroutine collimate_do_collimator(stracki)
   end if
 
 !++  Write beam ellipse at selected collimator
-  if (((cdb_cNameUC(icoll).eq.name_sel(1:mNameLen)) .or. (cdb_cName(icoll).eq.name_sel(1:mNameLen))) .and. do_select) then
-    do j = 1, napx
-      write(coll_ellipseUnit,'(1X,I8,6(1X,E15.7),3(1X,I4,1X,I4))') ipart(j),xv1(j), xv2(j), yv1(j), yv2(j), &
-     &        ejv(j), sigmv(j),iturn,secondary(j)+tertiary(j)+other(j)+scatterhit(j),nabs_type(j)
+  if(chr_toLower(cdb_cName(icoll)) == name_sel .and. do_select) then
```

Comment:
shouldn't we lower `name_sel` instead of `cdb_cName(icoll)`? I think that the latter is already in lower case...

Reply:
This is done so that the `name_sel` feature is not case sensitive. The `name_sel` variable is converted to lower case during input parsing, and here that must be done on the other side as well to ensure the comparison passes in case the `fort.2` / `collDB` has any upper case characters.

I had to do it that way since the old code required the upper case version, which means older input files will be upper case.

Reply:
Added a comment explaining the reason for this